### PR TITLE
Core Pool - Lens contracts documentation

### PR DESCRIPTION
**Pull request by defistar**
Core Pool - Lens contracts documentation
- ComptrollerLens
- VenusLens

### CODE REVIEW

File: core-pool/lens.md [hunk 1]
```diff
@@ -0,0 +1,668 @@
+# Core Pool lens
+
+## Introduction
+
+The Readonly functions in the lens contracts provide aggregated view of the vTokens, User Position, Liquidity and Snapshots of User Position in core-pool.
```
Comment (coreyar) [inline core-pool/lens.md]:
```suggestion
The read only functions in the lens contracts provide aggregated view of vTokens, user positions, and liquidity.
```

Comment (coreyar) [reply core-pool/lens.md]: Would be nice to have some descriptions about the functions provided. Could be inline with the auto generated docs

File: core-pool/lens.md [hunk 2]
```diff
@@ -0,0 +1,668 @@
...
+
+## Details
+
+- Lens contracts in core-pool are:
+
+1. ComptrollerLens
```
Comment (coreyar) [inline core-pool/lens.md]: We probably want these covered in their own files for the convinience of the reader

Comment (defistar) [reply core-pool/lens.md]: @coreyar shall i split this in to two:
comptroller-lens
venus-lens

File: core-pool/lens.md (hunk 1 above)
Comment (defistar) [reply core-pool/lens.md]: @coreyar 
means list of the functions in beginning?

Comment (coreyar) [reply core-pool/lens.md]: Rather than a list I was thinking about adding a short description to provide context on the function

Comment (defistar) [reply core-pool/lens.md]: @coreyar added context for each function below its signature
should i make it more detailed?

Comment (coreyar) [reply core-pool/lens.md]: yes that works too

File: core-pool/lens.md (hunk 2 above)
Comment (coreyar) [reply core-pool/lens.md]: yes

Comment (defistar) [reply core-pool/lens.md]: @coreyar splitting now. will push ASAP

File: core-pool/lens.md (hunk 1 above)
Comment (defistar) [reply core-pool/lens.md]: @coreyar added context with details below each function

File: core-pool/comptroller-lens.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
+# Core Pool lens
```
Comment (coreyar) [inline core-pool/comptroller-lens.md]:
```suggestion
# ComptrollerLens
```

File: core-pool/comptroller-lens.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+- ComptrollerLens contract has functions to get view in to the number of tokens can be seized on liquidation and hypothetical liquidity and shortfall of a user-account.
```
Comment (coreyar) [inline core-pool/comptroller-lens.md]:
```suggestion
The ComptrollerLens contract has functions to get the number of tokens that can be seized through liquidation, hypothetical account liquidity and shortfall of an account.
```

File: core-pool/venus-lens.md [hunk 5]
```diff
@@ -0,0 +1,553 @@
+# Core Pool lens
```
Comment (coreyar) [inline core-pool/venus-lens.md]:
```suggestion
# VenusLens
```

File: core-pool/venus-lens.md [hunk 6]
```diff
@@ -0,0 +1,553 @@
...
+    - tokenBalances of a specific vToken for a userAccount
+    - tokenBalances of all vTokens for a userAccount
+    - underlyingPrice of a vToken
+    - underlyingPriceInfo for a set of vTokens
+    - getAccountLimits (liquidity and shortfall) of a userAccount
+    - get Governance receipts of a user with votingBalance
+    - get Governance Proposal details of a list of proposalIds
+    - get XVSBalance details of a user Account
+    - get VotingBalance of a User Account
+    - get Pending XVS Rewards of a user Account
```
Comment (coreyar) [inline core-pool/venus-lens.md]:
```suggestion

The read only functions on the VenusLens contract provide a view into:
    - metadata of vToken
    -  daily XVS rewards for an account
    - account balance for a single vToken
    - account balances for all vTokens in an account
    - underlying price of a vToken
    - underlying prices for a set of vTokens
    - get liquidity and shortfall of an account
    - get user's vote history
    - get proposal details for a set of proposals
    - get account XVS balance, total votes, and delegated votes
    - get historical voting balance for a user
    - get pending XVS Rewards for an account
```
```
- [pendingVenus](#pendingVenus-info): get pending XVS Rewards for an account
```

**Pull request merged**